pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerMetrics.java

```diff
@@ -26,17 +26,25 @@
  *
  */
 public class BrokerMetrics extends AbstractMetrics<BrokerQueryPhase, BrokerMeter, BrokerGauge, BrokerTimer> {
+
+  public static final String METRICS_PREFIX_DEFAULT = "pinot.broker.";
```

Comment:
Can you add these constants into CommonConstants? Rename to DEFAULT_METRICS_PREFIX and DEFAULT_GLOBAL. Same for other classes